yamui.c

```diff
@@ -1483,7 +1484,11 @@ main(int argc, char *argv[])
 			app_systemd_notify = true;
 			break;
 		case 'd':
-			debugging = true;
+			log_debug("valgrind support enabled");
```

Comment:
Note that "valgrind support" = make an attempt to release dynamic memory to reduce noise in possible leaks logging -> the  "do_cleanup == true" case. And here it gets associated with "do_cleanup = false" - which is the wrong way around.

Reply:
Ok so that was the point. Yeah, overlooked this and *poof* now it is gone.